rust/repositories.bzl

```diff
@@ -61,6 +61,13 @@ def rust_repositories(
         type = "zip",
     )
 
+    maybe(
+        http_archive,
+        name = "rules_python",
+        url = "https://github.com/bazelbuild/rules_python/releases/download/0.1.0/rules_python-0.1.0.tar.gz",
+        sha256 = "b6d46438523a3ec0f3cead544190ee13223a52f6a6765a29eae7b7cc24cc83a0",
+    )
```

Comment:
I've run into a lot of problems with rules_python around python versioning and package dependencies.  While it's use in this PR is fairly contained, it does feel like a pretty heavy dependency to bring in.

Reply:
Noted. @UebelAndre and I discussed this a bit in #472. I agree that the
Pip side of `rules_python` is unorthodox, but I figured that this might
be okay since we only need the standard library. Does that seem okay to
you? If not, do you have an alternative suggestion for a toolchain to
handle #471 and #472?

Reply:
Looking at the `dir_zipper` it seems like that could be implemented in Rust using just `std`. The only thing I feel made sense to author in python was the webserver for hosting docs. I don't think `rules_python` is necessary for the goal of this pull request.

Reply:
I would agree with @UebelAndre here in that I think rust gives enough support here to handle this.  In general, unless it's particularly prohibitive, it's generally good to keep rules_rust as self-sufficient as possible.  (It's also easier to assume that people will be more familiar with rust/starlark when working within this project)

I think there is some precedent for this with the cargo_build_script_runner
https://github.com/bazelbuild/rules_rust/blob/master/cargo/cargo_build_script_runner/bin.rs

Reply:
Okay, sure: rewritten as a Rust script. (Also wins better handling for
non-UTF-8 path names; loses nice arg parsing and tests, since we don’t
have a stdlib `zipfile` equivalent.)